packit_service/models.py

```diff
@@ -1269,18 +1426,8 @@ def create(
             build.scratch = scratch
             session.add(build)
 
-            if run_model.koji_build:
-                # Clone run model
-                new_run_model = PipelineModel.create(
-                    type=run_model.job_trigger.type,
-                    trigger_id=run_model.job_trigger.trigger_id,
-                )
-                new_run_model.srpm_build = run_model.srpm_build
-                new_run_model.koji_build = build
-                session.add(new_run_model)
-            else:
-                run_model.koji_build = build
-                session.add(run_model)
+            koji_build_group.koji_build_targets.append(build)
+            session.add(koji_build_group)
```

Comment:
nicely simplified

Reply:
Just moved elsewhere, to be honest, the duplicating logic is still necessary inside group :D

Reply:
noticed after I commented :D but still, this method is simpler